companion/HELP.md

```diff
@@ -0,0 +1,167 @@
+## Blackmagic Design AudioMonitor 12G
+
+This module will connect to the Blackmagic Design AudioMonitor 12G device.
+
+![Sample BMD AudioMonitor Layout](./images/layout01.png)
+
+![Sample BMD AudioMonitor Layout](./images/layout02.png)
+
+#### Version 3
+
+This module is only compatible with version 3 of Bitfocus Companion and above. If you have used a previous version of the module, it is easy to build a new one very quickly as it is feature complete with Actions, Feedback and multiple Presets.
+
+#### Configuration
+
+- Enter the IP address for your device. ie. 192.168.10.50
+- Enter the port address for your device. The default port for the device is 9996.
+
+#### Actions
+
+The following actions are available:
+
+* **Audio Meter**
+
+    This can be changed to: VU (-20dBFS Ref), VU (-18dBFS Ref), PPM EBU (-20dBFS Ref), PPM EBU (-18dBFS Ref), PPM BBC (-20dBFS Ref), PPM BBC (-18dBFS Ref), Loudness (EBU +9scale) or Loudness (EBU +18 scale).
+
+* **Audio Input**
+
+    This can be changed to: SDI Stereo 3-4, SDI Stereo 3-4, SDI Stereo 5-6, SDI Stereo 7-8, SDI Stereo 9-10, SDI Stereo 11-12, SDI Stereo 13-14, SDI Stereo 15-16, XLR AES/EBU Stereo 1-2, XLR Analog Stereo or RCA Stereo.
```

Comment:
Typo:
```suggestion
    This can be changed to: SDI Stereo 1-2, SDI Stereo 3-4, SDI Stereo 5-6, SDI Stereo 7-8, SDI Stereo 9-10, SDI Stereo 11-12, SDI Stereo 13-14, SDI Stereo 15-16, XLR AES/EBU Stereo 1-2, XLR Analog Stereo or RCA Stereo.
```